commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -266,8 +270,22 @@ def get_orders(self, customer: CTCustomer, offset=0,
             offset=offset,
             expand=list(expand)
         )
+        end_time = datetime.datetime.now()
+        logger.info(
+            "[UserOrdersView] Get CT orders query call finished at %s with total duration: %s",
+            end_time, end_time - start_time
+        )
```

Comment:
For our easiness, we can simply print the datetime difference in seconds:

```suggestion
        logger.info(
            "[UserOrdersView] Get CT orders query call finished at %s with total duration: %s",
            end_time, (end_time - start_time).total_seconds()
        )
```

Please make this improvement wherever needed

Reply:
Done